supervision/dataset/formats/coco.py

```diff
@@ -102,15 +105,29 @@ def coco_annotations_to_detections(
     xyxy = np.asarray(xyxy)
     xyxy[:, 2:4] += xyxy[:, 0:2]
 
+    if use_iscrowd:
+        iscrowd = [
+            image_annotation["iscrowd"] for image_annotation in image_annotations
+        ]
+        area = [image_annotation["area"] for image_annotation in image_annotations]
+        data = dict(
+            iscrowd=np.asarray(iscrowd, dtype=int), area=np.asarray(area, dtype=float)
+        )
+    else:
+        iscrowd = [0] * len(image_annotations)
+        area = None
```

Comment:
It looks like we’re not actually using `iscrowd` or `area`; we’re just returning empty dictionaries. I think it makes sense to remove both `iscrowd` and `area`.